Modules/Core/Controllers/ReportsController.php

```
@@ -37,79 +53,102 @@ public function salesByClient(Request $request): \Illuminate\Contracts\View\View
      *
-     * @originalFile ReportsController.php
+     * @legacy-function invoiceAging
+     * @legacy-file application/modules/reports/controllers/Reports.php
      */
     public function invoiceAging(Request $request): \Illuminate\Contracts\View\View
     {
         if ($request->input('btn_submit')) {
-            $results = (new ReportsService())->invoiceAging();
-            $data    = [
+            $results = $this->reportsService->invoiceAging();
+            $data = [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (undefined)

(UnusedLocalVariable)